ci/script.sh

```diff
@@ -2,7 +2,7 @@
 
 set -ex
 
-cargo build --all --verbose
+cargo build --verbose
```

Comment:
How old? Since we're going to do a semver bump anyway, we can increase the minimum Rust version required.

Reply:
It's probably not 100% necessary to bump the minimum Rust version, because we already run a `cargo test --all --verbose` further down, once we've established that we have a reasonably new Rust version. So test coverage has actually gone up from before.